### PR TITLE
Update ToDo patterns in dispatch.lu

### PR DESCRIPTION
**Pull request by feich-ms**
Update ToDo patterns in dispatch.lu
## Description
Last time, I updated the luis model in skill project and forgot to update the patterns in VA manually. This PR is to update ToDo patterns in dispatch.lu

## Related Issue
#51 

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate unit tests
- [ ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
- [ ] A duplicate issue is filed to track future  work.

If you have updated responses or `.lu` files:
- [x] All languages have been updated
- [x] You have tested deployment with your new models

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/zh-hans/dispatch.lu [hunk 1]
```diff
@@ -12,101 +12,85 @@
...
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#FindCalendarWho)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#FindDuration)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#FindMeetingRoom)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#GoBack)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#NextMeeting)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#NoLocation)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#ReadAloud)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#Summary)
-- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/en/calendar.lu#TimeRemaining)
+- [l_Calendar](../../../../skills/calendarskill/CognitiveModels/LUIS/zh-hans/calendar.lu#ChangeCalendarEntry)
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/zh-hans/dispatch.lu]: This folder has changed from "zh-hans" to "zh" in the latest LU bug fix

File: solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu [hunk 2]
```diff
@@ -49,61 +49,48 @@
...
-- contrassegnare l'attività  {TaskContent} come completato
-- attività Mark  {TaskContent} come completato
-- contrassegnare l'attività  {TaskContent} come fatto
-- Mark  {TaskContent} come completo
-- Mark  {TaskContent} come fatto
-- completare l'attività  {TaskContent} 
-- contrassegnare l'attività  {TaskContent} come completo
-- attività Mark  {TaskContent} come fatto
-- attività Mark  {TaskContent} come finito
+- aggiungere un'attività al negozio  {ShopContent}
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu]: You'll probably have conflicts with master now. It looks good, but just make sure there are no duplicates in the updated files. You can use this powershell script to see if there are any dupes:

```
Get-Content <file_name.lu> | Group-Object | Where-Object { $_.Count -gt 1 } | Select -ExpandProperty Name
```

File: solutions/Virtual-Assistant/src/csharp/assistant/DeploymentScripts/zh-hans/update_zh.bat [hunk 3]
```diff
@@ -7,6 +7,6 @@ call ludown parse toluis --in %0\..\..\..\..\skills\emailskill\CognitiveModels\L
 call ludown parse toluis --in %0\..\..\..\..\skills\todoskill\CognitiveModels\LUIS\zh-hans\todo.lu -o %0\..\..\..\DeploymentScripts\zh-hans --out todo.luis -n ToDo
 call ludown parse toluis --in %0\..\..\..\..\skills\pointofinterestskill\CognitiveModels\LUIS\zh-hans\pointofinterest.lu -o %0\..\..\..\DeploymentScripts\zh-hans --out pointofinterest.luis -n PointOfInterest
 call ludown parse toqna  --in %0\..\..\..\CognitiveModels\QnA\zh-hans\faq.lu -o %0\..\..\..\DeploymentScripts\zh-hans -n faq.qna
-call ludown parse toluis --in CognitiveModels\LUIS\zh-hans\dispatch.lu -o %0\..\..\..\DeploymentScripts\zh-hans --out dispatch.luis -n Dispatch -i Dispatch -c zh-cn
+call ludown parse toluis --in %0\..\..\..\CognitiveModels\LUIS\zh-hans\dispatch.lu -o %0\..\..\..\DeploymentScripts\zh-hans --out dispatch.luis -n Dispatch -i Dispatch
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/assistant/DeploymentScripts/zh-hans/update_zh.bat]: The latest in master should fix this issue and have added culture for each file

**Review by lauren-mills — CHANGES_REQUESTED** [review]

**Comment by feich-ms** [comment]
@lauren-mills Close this PR due to merge chaos and create a new one here https://github.com/Microsoft/AI/pull/364. Please review that one.

File: solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu (hunk 2 above)
Comment (feich-ms) [reply solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu]: Tested and no duplicated patterns are there.

File: solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/zh-hans/dispatch.lu (hunk 1 above)
Comment (feich-ms) [reply solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/zh-hans/dispatch.lu]: yes, have removed the folder of zh-hans. Please see PR #364

File: solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu (hunk 2 above)
Comment (feich-ms) [reply solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels/LUIS/it/dispatch.lu]: Please see PR #364

File: solutions/Virtual-Assistant/src/csharp/assistant/DeploymentScripts/zh-hans/update_zh.bat (hunk 3 above)
Comment (feich-ms) [reply solutions/Virtual-Assistant/src/csharp/assistant/DeploymentScripts/zh-hans/update_zh.bat]: yes, thx. Please see PR #364

**Pull request closed without merging**